stdlib/public/core/Substring.swift

```diff
@@ -200,7 +200,6 @@ extension Substring: StringProtocol {
     return _slice.distance(from: start, to: end)
   }
 
-  @inlinable
   public subscript(i: Index) -> Character {
     @inline(__always) get { return _slice[i] }
```

Comment:
So this one remains inlinable? I am guessing since it's __always it did have some measurable benefit.

Reply:
inline(always) does not imply inlinable.
In this case it's inline(always) within the stdlib. But yes, the question is, if we still need this inline(always).